site/src/components/Navbar.tsx

```diff
@@ -362,9 +362,12 @@ export const Navbar: VFC<NavbarProps> = ({
                 <IconButton
                   onClick={() => setDisplayMobileNav(!displayMobileNav)}
                   sx={{
-                    "& svg": isNavbarDark
-                      ? { color: theme.palette.purple.subtle }
-                      : {},
+                    "& svg": {
+                      fontSize: 27.5,
+                      ...(isNavbarDark
+                        ? { color: theme.palette.purple.subtle }
+                        : {}),
+                    },
```

Comment:
Doesn't need to be changed, but I think this can just be passed to the FontAwesomeIcon `sx` prop directly instead of using `"& svg": {`

Reply:
I agree. Added here => b6c2efb